servers/camera/camera_server.h

```diff
@@ -111,6 +111,12 @@ class CameraServer : public Object {
 	// Intended for use with custom CameraServer implementation.
 	RID feed_texture(int p_id, FeedImage p_texture);
 
+	// Platform lifecycle callbacks (virtual, default empty implementation).
+	virtual void handle_application_pause() {}
+	virtual void handle_application_resume() {}
+	virtual void handle_display_rotation_change() {}
+	virtual void handle_display_rotation_change(int p_orientation) { (void)p_orientation; }
```

Comment:
Seems like we can remove `handle_display_rotation_change()` and update the Android implementation to also pass the new `p_orientation`.